events.ts

```diff
@@ -36,9 +50,19 @@ const cover = {
     if (wrapper) {
       if (options.init) {
         if ((selectorElement = wrapper.querySelector(selector))) {
```

Comment:
I'm using this now. Because push will add an empty object when query is not found.

```
      if (options.querySelectorAll) {
        elements = elements.concat(
          Array.from(wrapper.querySelectorAll(selector))
        );
      } else {
        let selectorElement = wrapper.querySelector(selector);
        if (selectorElement) {
          elements.push(selectorElement);
        }
      }
```